src/Surfnet/Stepup/Identity/Entity/VerifiedSecondFactor.php

```diff
@@ -138,8 +139,26 @@ public function canBeVettedNow()
         );
     }
 
-    public function vet(DocumentNumber $documentNumber)
+    public function vet(DocumentNumber $documentNumber, $provePossessionSkipped)
     {
+        if ($provePossessionSkipped) {
+            $this->apply(
+                new SecondFactorVettedPossessionSkippedEvent(
```

Comment:
I see the logic in not adding a flag to the existing SecondFactorVettedEvent as this is a clearly different variety of a regularly vetted second factor. However I find the name of this event rather indistinct. I do not directly have a more fitting proposal, but this might be something we want to discuss with our PO as this kind of naming should match our ubiquitous language.

Reply:
We could for example use:
`SecondFactorVettedAndTokenPossessionSkipped`
`SecondFactorVettedWithoutTokenProveOfPossession`
But currently I'm ok with using the initial name...

Reply:
I like `SecondFactorVettedWithoutTokenProveOfPossession`

However mind that this contains a typo, should be:

`SecondFactorVettedWithoutTokenProofOfPossession`

Reply:
Agree, will change that